changelogs/22.04

```diff
@@ -0,0 +1,52 @@
+* Pop!_OS is now based on Ubuntu 22.04.
+
+* COSMIC
+** GNOME 42 support has been added for COSMIC.
+** The Shell Theme for GNOME 42 has been updated.
+** Icons are now SVG-based instead of PNG-based.
+** Dark and light modes can have independent backgrounds.
+** Settings for Desktop Options, Background, Appearance, Dock, and Workspaces are now searchable in the Launcher.
+** The Pop robot face has been added as the default avatar.
+** Improvements to cosmic-workspaces:
+    ** An issue with the location of the Workspace picker on secondary displays has been fixed.
+    ** An issue with the scaling of the Workspace picker on displays of varying resolutions has been fixed.
+    ** Performance has been improved.
+    ** An issue with alignment on HiDPI displays has been fixed.
+    ** The OSD picker is now smaller.
+** COSMIC logging has been limited, resulting in increased system performance.
+
+* Pop!_OS System Updater
+** New options for installing updates are now available in Settings > OS Upgrade and Recovery.
+    ** To reduce distractions, update notifications are now set to show weekly and can be set to daily or monthly.
+    ** Updates can be scheduled and installed automatically (includes Debian, Flatpak, and *nix packages).
+** The Pop upgrade service will now only be active when checking for release upgrades and doing a release upgrade, instead of being active 24/7.
+** Pop upgrade Debian package downloads are now resumable.
+
+* Pop!_Shop
+** Installed NVIDIA drivers are now visible in the Pop!_Shop and older drivers are available to install.
+** The Pop!_Shop is now uninstallable, and the GNOME Software Center can be installed.
+
+* Packaged Applications
+** Debian packaging has been added for Firefox.
+** PipeWire replaces the PulseAudio audio system.
+** Mesa 22 has been added.
+** Irqbalance has been removed from the default installation.
+** Rust is now packaged in the default installation, and will be kept up-to-date with upstream releases.
```

Comment:
This one should be `Rust is now packaged in the Pop repositories`, since it's not part of any install.

Reply:
Or left out entirely since it's only relevant to Rust programmers.